modules/services/safeeyes.nix

```
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.safeeyes;
+
+in {
+  meta.maintainers = [ maintainers.rosuavio ];
```

When referring to a maintainer in the set defined inside this repo you need to reach inside the `hm` library:

```suggestion
  meta.maintainers = [ hm.maintainers.rosuavio ];
```